index.js

```diff
@@ -1,15 +1,23 @@
-require('./lib/array_fill');
-require('./lib/array_intersect');
-require('./lib/array_qsort');
-require('./lib/array_unique');
-require('./lib/fs_scandir');
-require('./lib/object_watch');
-require('./lib/string_format');
-require('./lib/string_hashcode');
-require('./lib/string_pipe');
+module.exports = function (opts) {
+  opts = opts || {};
```

Comment:
Non-0 numbers, strings & `true` can still slip through this check.
In order to prevent unexpected behavior later, it could be replaced by the following:

```js
// Non-object is invalid here.
// To catch null: opts || {}
opts = ( 'object' === typeof opts ) ? ( opts || {} ) : {};
```